apps/web/src/views/PositionManagers/containers/VaultCards.tsx

```diff
@@ -16,6 +16,12 @@ import {
 } from '../hooks'
 import { ThirdPartyVaultCard } from './PCSVaultCard'
 
+const createSearchRegex = (wordText) => {
+  const words = wordText.split(/\s+/)
+  const escapedWords = words.map((word) => word.replace(/[.*+?^${}()|[\]\\]/g, '\\$&'))
+  const pattern = escapedWords.map((word) => `(?=.*\\b${word}\\b)`).join('')
```

Comment:
Can we support partial search? For example I'd prefer to be able to search `bnb` instead of typing `wbnb`